docs/sharing.rst

```diff
@@ -312,13 +312,18 @@ Azure Repos credentials
 
 Nextflow has a builtin support for `Azure Repos <https://azure.microsoft.com/en-us/services/devops/repos/>`_, a Git source
 code management service hosted in the Azure cloud. To access your Azure Repos with Nextflow provide the repository credentials
-using the configuration snippet shown below:
+using the configuration snippet shown below::
 
     providers {
 
         azurerepos {
-            user = 'your-user-name'
-            password = 'your-personal-access-token'
+            server = 'https://dev.azure.com'
+            endpoint = 'https://dev.azure.com' // Can be omitted when it has the same value as the server field.
```

Comment:
> I'd stick to endpoint, as it's guaranteed whether you set a custom endpoint or not

Just to confirm, did you mean `I'd stick to *server*` ?

Reply:
In fact I meant `stick to endpoint`. In case the user sets an endpoint in the scm file, the code will use it. Otherwise, it is set to the server value. However, in case, the user sets the server instead of the endpoint, it still is consistently used but the endpoint has prevalence over the server. Given that the Azure service is pure cloud at this time, I'm not sure it really matters.